docs/ref/remote-cli.md

```diff
@@ -69,7 +69,7 @@ headscale apikeys expire --prefix "<PREFIX>"
 
         !!! bug
 
-            Headscale 0.23.0 requires at least an empty configuration file when environment variables are used to
+            Headscale 0.23.0/0.24.0 requires at least an empty configuration file when environment variables are used to
```

Comment:
Is this something that can be made a variable with the new version thing?

Reply:
In this case no because its specific for those versions.

In general: the `extra.headscale.version` in `mkdocs.yml` should by dynamic in the future, but its not finished yet.